tests/5.1/teams/test_target_set_num_teams.c

```diff
@@ -0,0 +1,45 @@
+//===--------------------- test_target_set_num_teams.c ----------------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+//
+// This test uses the omp_set_num_teams utility to set the default number of
+// teams allocated for team clauses. The test runs teams and checks to see if
+// the number allocated is correct and reflective of the new default set. It
+// then checks to see if an explicitly stated num_teams clause overrides
+// the omp_set_num_teams properly.
+//
+//===------------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+int main() {
+
+
+	int errors = 0; 
+	int num_teams = 0;
+	omp_set_num_teams(4);
+        	
+	#pragma omp teams 
+	{
+		if (omp_get_team_num() == 0 ) {
+			num_teams = omp_get_num_teams();
+		}
+	}               
+	
+	OMPVV_ERROR_IF(num_teams != 4, "Number of teams detected was not the number set by omp_set_num_teams()");
```

Comment:
```suggestion
	OMPVV_ERROR_IF(num_teams <= 4, "Number of teams detected was not the number set by omp_set_num_teams()");
```
pg. 101 5.1 spec
21 If the num_teams clause is not specified and the value of the nteams-var ICV is greater than zero,
22 the number of teams created is less or equal to the value of the nteams-var ICV. Otherwise, the
23 number of teams created is implementation defined, but it will be greater than or equal to 1.
The nteams-var is greater than zero, as such it is allowed for the number of teams created to be less than or equal to the ICV value.